src/PowerShellEditorServices/Services/PowerShell/Host/PsesInternalHost.cs

```diff
@@ -29,11 +29,11 @@ internal class PsesInternalHost : PSHost, IHostSupportsInteractiveSession, IRuns
     {
         private const string DefaultPrompt = "PSIC> ";
         // This is a default that can be overriden at runtime by the user or tests.
-        private static string s_bundledModulePath = Path.GetFullPath(Path.Combine(
+        private static string _bundledModulePath = Path.GetFullPath(Path.Combine(
```

Comment:
Any particular reason for this? I usually stick with [dotnet/runtime's style guide](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md). I believe that's what PowerShell/PowerShell uses as well.

Reply:
Roslyn Analyzer was complaining, but then it kept complaining!

Reply:
Reverted.